migrations/sql/V2018.02.20.0900__AddSmallCountryColumnToBurdenEstimate.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE burden_estimate
+  ADD COLUMN country_nid SMALLINT NULL;
```

Comment:
As with stochastic, once API is updated we should remove `country` column, populate this column, recreate the uniqueness constraint using this new field and remove the old column.